modules/meeting/app/components/meeting_agenda_items/item_component/show_component.rb

```diff
@@ -239,5 +247,13 @@ def in_backlog?
     def in_template?
       @meeting.templated?
     end
+
+    def note_or_outcome_action_added?
+      (@meeting_agenda_item.editable? && @meeting_agenda_item.notes.blank?) || add_outcome_action?
+    end
+
+    def move_to_different_section_or_meeting_action_added?
+      (!in_template? || (in_template? && in_backlog?)) || move_to_next_meeting_enabled?
+    end
```

Comment:
```suggestion
    def move_to_different_section_or_meeting_action_added?
      (!in_template? || in_backlog?) || move_to_next_meeting_enabled?
    end
```

the `in_template?` in the second part of `(!in_template? || (in_template? && in_backlog?))` is either never evaluated or always true, so it can be removed.

And so the parens can be removed as well

```suggestion
    def move_to_different_section_or_meeting_action_added?
      !in_template? || in_backlog? || move_to_next_meeting_enabled?
    end
```